src/js/view/controls/components/timeslider.js

```diff
@@ -177,7 +177,7 @@ class TimeSlider extends Slider {
             return;
         }
         this.reset();
-        model.set('cues', []);
+        this.updateCues(model, model.get('cues'));
```

Comment:
May not need this because `updateCues()` is called whenever the playlistItem on the model is changed.

Reply:
Nevermind, the cues are removed from the UI when entering ads mode.